lizmap/modules/admin/controllers/logs.classic.php

```diff
@@ -252,12 +258,16 @@ public function eraseError()
         $rep = $this->getResponse('redirect');
 
         // Erase the log file
-        try {
-            $logPath = jApp::varPath('log/errors.log');
-            jFile::write($logPath, '');
-            jMessage::add(jLocale::get('admin~admin.logs.error.file.erase.ok', array('log_detail')));
-        } catch (Exception $e) {
-            jLog::log('Error while emptying the error log file');
+        if (\jAcl2::check('lizmap.admin.lizmap.log.delete')) {
```

Comment:
you could indicate this rights on the `$pluginParams` directly like the exemple for emptyCounter